doc/api/errors.md

```diff
@@ -1640,6 +1652,13 @@ A non-context-aware native addon was loaded in a process that disallows them.
 
 A given value is out of the accepted range.
 
+<a id="ERR_PKG_PATH_NOT_EXPORTED"></a>
+### `ERR_PKG_PATH_NOT_EXPORTED`
```

Comment:
Can we please spell this out? `ERR_PACKAGE_PATH_NOT_EXPORTED`. I see no reason to abbreviate “package.”

Reply:
Changed.